### DISCUSSION TITLE
DS%40UPenn/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#50] [Announcements]
# DS%40UPenn/

https://csmsapp.github.io/DS%40UPenn/

**Comment by Patrick-ChenKehan** [comment]
这个项目的项目质量和难度匹配吗？

**Reply by xichenpan** [reply]
我觉得虽然不如mcis的质量和难度差距大，总的来说质量还是不如同档项目的

**Comment by FeijiangHan** [comment]
陆本末九皆有录取，可以放心冲。
感觉会看中课外经历和文书
